src/cli/snapshots/pixi__cli__completion__tests__nushell_completion.snap

```diff
@@ -2,22 +2,13 @@
 source: src/cli/completion.rs
 expression: result
 ---
-  
-  def "nu-complete pixi run" [] {
-    ^pixi info --json | from json | get environments_info | get tasks | flatten | uniq
-  }
-
-  def "nu-complete pixi run environment" [] {
-    ^pixi info --json | from json | get environments_info | get name
-  }
-
   # Runs task in project
   export extern "pixi run" [
-    ...task: string@"nu-complete pixi run"           # The pixi task or a task shell command you want to run in the project's environment, which can be an executable in the environment's PATH
+    ...task: string           # The pixi task or a task shell command you want to run in the project's environment, which can be an executable in the environment's PATH
     --manifest-path: string   # The path to `pixi.toml` or `pyproject.toml`
     --frozen                  # Install the environment as defined in the lockfile, doesn't update lockfile if it isn't up-to-date with the manifest file
     --locked                  # Check if lockfile is up-to-date before installing the environment, aborts when lockfile isn't up-to-date with the manifest file
-    --environment(-e): string@"nu-complete pixi run environment" # The environment to run the task in
+    --environment(-e): string@"nu-complete pixi run" # The environment to run the task in
```

Comment:
The original was correct. Did you modify this by accident?

Reply:
Ack. This flipped back and forth a few times in squashes and maintaining this patch through several releases. The test isn't testing really what comes from clap but that the regex replace works as expected. That still gets tested correctly even if the input isn't exactly what comes from clap.

Reply:
> That still gets tested correctly even if the input isn't exactly what comes from clap.

Yeah, makes sense. Still, let's use exactly what comes from clap :)